src/Core/Cache/DefaultCacheFactory.php

```diff
@@ -114,20 +117,52 @@ protected function isPHPFilesSupported()
     }
 
     /**
-     * @param string $class
-     * @param array $args
-     * @return CacheInterface
+     * Creates an object with a PSR-16 interface, usually from a PSR-6 class name
+     *
+     * Quick explanation of caching standards:
+     * - Symfony cache implements the PSR-6 standard
+     * - Symfony provides adapters wrap a PSR-6 backend with a PSR-16 interface
+     * - Silverstripe uses the PSR-16 to interface with caches
```

Comment:
```suggestion
     * - Silverstripe uses the PSR-16 adapters to interact with caches. It does not directly interact with the PSR-6 classes
```
I _think_ this is what was meant?

Reply:
Updated

Reply:
It still says "to interface with caches" - did you mean interact? I'd like to avoid confusion with the php concept of interfaces vs the actual interaction.

Reply:
It still says "to interface with caches" - did you mean interact? I'd like to avoid confusion with the php concept of interfaces vs the actual interaction.
(sorry about the double notification - tried to get this added to my pending review instead of commenting immediately but it won't let me)

Reply:
Updated